homeassistant/components/enocean/sensor.py

```diff
@@ -281,3 +287,43 @@ def value_changed(self, packet):
             self._attr_native_value = "tilt"
 
         self.schedule_update_ha_state()
+
+
+class EnOceanWallSwitch(EnOceanSensor):
+    """Representation of an EnOcean wall switch.
+    EEPs (EnOcean Equipment Profiles):
+    - F6-02-00 
+    """
+
+    def value_changed(self, packet):
+        """Update the internal state of the sensor."""
+        action = packet.data[1]
+
+        if action == 0xF0:
```

Comment:
This integration has existing code with protocol details that belong in a 3rd party library. We won't allow changes or additions that add more protocol details to the integration. The details need to be extracted to a 3rd party library published on PyPI first.

Reply:
Why the windows handle are integrate like this but not the wall switch??? they are in the same family (rorg f6). I am not the first user how would like to see it in the dashboard.
So i don't understand your answers.
i am new user of HA, so what is Pypi? 
i just not understand your answer
Best regards
![image](https://user-images.githubusercontent.com/123359286/214309968-2b311ba6-a013-4058-8c7b-bd84c6be54b8.png)
[EnOcean_Equipment_Profiles_EEP_V2.6.2_public.pdf](https://github.com/home-assistant/core/files/10490488/EnOcean_Equipment_Profiles_EEP_V2.6.2_public.pdf)

Reply:
I tried to explain this above:

> This integration has existing code with protocol details that belong in a 3rd party library. We won't allow changes or additions that add more protocol details to the integration.

The existing code is eg the window sensors you mention.